docker-compose.yml

```
@@ -0,0 +1,18 @@
+---
+version: '3.8'
+services:
+ 
+   semantic-sh:
```

Providing commandline parameters to server.py by defining environment variables in docker-compose will give more configurable docker usage.  I'm not sure the code I write below will work properly, please test before committing.
```suggestion
   semantic-sh:
     environment:
    - model_type
    - model_path
    - key_size
    - dim
    - host
    - port
```

---

Thanks for the comment, shall I create an .env file ? 
rather than using hard coded values in the docker-compose.yml file.

---

What are your recommended/default values for ?  
```txt
--key-size KEY_SIZE   Hash length in bits
--dim DIM             Dimension of text representations according to chosen model type
```

---

Yes, it'd probably better.

---

That's what I did, in my next push :-)